applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -111,6 +114,12 @@ void rsp_send(const uint8_t *str, size_t len)
 	(void)uart_send(str, len);
 }
 
+void datamode_send(const uint8_t *data, size_t len)
```

Comment:
I'm a bit confused, where/how is this new function used?

Reply:
Sorry to be later. This API is prepared for FTP download as well as TCP/UDP client RX while in data mode. Will be used in the following PRs.